docs/running/index.rst

```diff
@@ -64,7 +64,41 @@ The logs can also be found easily in Podman Desktop by clicking on a specific co
 Debugging the Development Application
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To debug the application, `view the setup in the DEVELOPERS.md file <https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/blob/master/DEVELOPERS.md>`_.
+To debug the application, `view the setup in the DEVELOPERS.md file
+<https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/blob/master/DEVELOPERS.md>`_.
+
+Resetting the Database
+^^^^^^^^^^^^^^^^^^^^^^
+
+During development, you may need to reset the database to a clean state. You can do this with the
+following command:
+
+.. code-block:: bash
+
+    # Using Docker:
+    docker compose -f compose.dev.yml exec app python manage.py reset
+
+    # Using Podman:
+    podman-compose -f compose.dev.yml exec app python manage.py reset
+
+
+This will prompt you to confirm the deletion of all data in the database. Type "yes" to proceed.
```

Comment:
If you type "yes" the database won't be reset, should say "y"

```suggestion
This will prompt you to confirm the deletion of all data in the database. Type "y" to proceed.
```